packages/neon/neon/lib/src/utils/push_utils.dart

```diff
@@ -132,8 +132,7 @@ class PushUtils {
               final image = recorder.endRecording().toImageSync(scaledSize.width.toInt(), scaledSize.height.toInt());
               final bytes = await image.toByteData(format: ImageByteFormat.png);
 
-              final bitmap = await Bitmap.fromProvider(MemoryImage(bytes!.buffer.asUint8List()));
-              largeIconBitmap = ByteArrayAndroidBitmap(bitmap.buildHeaded());
+              largeIconBitmap = ByteArrayAndroidBitmap(img.encodeBmp(img.decodePng(bytes!.buffer.asUint8List())!));
```

Comment:
I think this can be done in a more performant way.  Maybe:

```suggestion
final imageStuff = img.Image.fromBytes(
  width: scaledSize.width.toInt(),
  height: scaledSize.height.toInt(),
  bytes: bytes!.buffer,
  numChannels: 4,
);

largeIconBitmap = ByteArrayAndroidBitmap(img.encodeBmp(imageStuff));
```

I can further investigate this if needed

Reply:
Ah I see, I was already wondering if the png step was really necessary

Reply:
How is this more performant?

Reply:
I just thought it would be as we wouldn't need to encode and decode again.

I didn't test it yet

Reply:
But it still does the conversion from bytes to image? I'd think it's the same, but maybe your suggestion is more performant :woman_shrugging:

Reply:
Can we just merge this and optimize it later?